app/setupHandlers.go

```diff
@@ -12,11 +9,5 @@ func SetupHandlers(app *SifchainApp) {
 	app.UpgradeKeeper.SetUpgradeHandler("changePoolFormula", GetPoolChangeFunc(app))
 	app.UpgradeKeeper.SetUpgradeHandler("release-20210401000000", func(ctx sdk.Context, plan upgrade.Plan) {})
 	app.UpgradeKeeper.SetUpgradeHandler("release-20210407042000", GetAddDispensation(app))
-	SetState(app)
-}
-
-func SetState(app *SifchainApp) {
-	app.SetStoreLoader(bam.StoreLoaderWithUpgrade(&types.StoreUpgrades{
-		Added: []string{dispensation.ModuleName},
-	}))
+	app.UpgradeKeeper.SetUpgradeHandler("release-20210414000000", func(ctx sdk.Context, plan upgrade.Plan) {})
```

Comment:
What are you trying to do here? This PR should not be changing upgrade logic as it should only be around documentation.

Reply:
sorry, not sure how these commits got into this PR